packages/core/src/dev/activation.ts

```diff
@@ -435,6 +445,15 @@ async function expireSsoConnections() {
     )
 }
 
+export function forceQuitIde() {
+    // This current process is the ExtHost. Killing it will cause all the extensions to crash
+    // for the specific IDE instance.
+    // The PPID does not seem to be the IDE instance but the root VS Code process, killing
+    // this crashes all VS Code instances.
```

Comment:
PPID = parent PID. https://nodejs.org/api/process.html#processppid is this comment needed?

does it kill all vscode instances or only the current one?


```suggestion
    // The PPID (parent process) is vscode itself, killing it crashes all vscode instances.
```

Reply:
Killing the PPID kills all VS Code instances, not just the current one. After some testing there is a completely separate PID that kills the one specific VSC instance, but there isn't a reliable way to get it. We could maybe do PID+2 or something from some quick testing, but this is unreliable.